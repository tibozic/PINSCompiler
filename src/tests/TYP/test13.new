[1:43-1:48]: ERROR: Comparison operations only support integer and logical values
