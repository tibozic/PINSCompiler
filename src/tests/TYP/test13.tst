fun comp(a: string, b: string): logical = a < b
