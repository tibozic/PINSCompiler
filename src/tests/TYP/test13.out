[1:43-1:48]: ERROR: Comparison operator `LT` only supports integer and logical values

